test/global_uid_test.rb

```diff
@@ -255,6 +263,14 @@ class ParentSubclassSubclass < ParentSubclass
           CreateWithNoParams.down
         end
       end
+
+      if ActiveRecord::VERSION::MINOR >= 1
+        describe "has_and_belongs_to_many associations" do
+          it "inherits global_uid_disabled from the left-hand-side of the association" do
+            assert Account::HABTM_People.global_uid_disabled
+          end
+        end
+      end
     end
```

Comment:
Might be worth adding another test verifying that if the left-hand-side of the association has global_uid enabled, so does the HABTM?